src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -1189,6 +1189,125 @@ groovyscript.wiki.naturesaura.spawning.entity.value=Sets the entity spawned
+groovyscript.wiki.prodigytech.explosion_furnace.description=Uses an explosive, a dampener, and an optional reagent to convert items. The power value of all recipes, all explosives, and all dampeners should be close to avoid an efficiency loss.
+groovyscript.wiki.prodigytech.explosion_furnace.power.value=Sets the power used by this recipe.
+groovyscript.wiki.prodigytech.explosion_furnace.craftPerReagent.value=Sets the number of input items that can be converted with 1 reagent.
+
+groovyscript.wiki.prodigytech.explosion_furnace_additives.title=Explosion Furnace Additives
+groovyscript.wiki.prodigytech.explosion_furnace_additives.description=Turn an item into an explosive or into a dampener when inserted into the Explosion Furnace.
+groovyscript.wiki.prodigytech.explosion_furnace_additives.addDampener=Adds a dampener.
+groovyscript.wiki.prodigytech.explosion_furnace_additives.addExplosive=Adds an explosive.
+groovyscript.wiki.prodigytech.explosion_furnace_additives.removeAllDampeners=Removes all registered dampeners.
+groovyscript.wiki.prodigytech.explosion_furnace_additives.removeAllExplosives=Removes all registered explosives.
+groovyscript.wiki.prodigytech.explosion_furnace_additives.removeDampener=Removes a dampener by the input item.
+groovyscript.wiki.prodigytech.explosion_furnace_additives.removeExplosive=Removes an explosive by the input item.
+
+groovyscript.wiki.prodigytech.heat_sawmill.title=Heat Sawmill
+groovyscript.wiki.prodigytech.heat_sawmill.description=Wood processing machine with 1 input, 2 outputs and an optional chance for the 2nd output.
+groovyscript.wiki.prodigytech.heat_sawmill.time.value=The time needed to perform the recipe, in ticks. Will run faster than this number when heated over 80 C.
+groovyscript.wiki.prodigytech.heat_sawmill.secondaryChance.value=Sets the chance to get the 2nd output.
+
+groovyscript.wiki.prodigytech.magnetic_reassembler.title=Magnetic Reassembler
+groovyscript.wiki.prodigytech.magnetic_reassembler.description=A simple 1 to 1 processing machine for dusts.
+groovyscript.wiki.prodigytech.magnetic_reassembler.time.value=The time needed to perform the recipe, in ticks. Will run faster than this number when heated over 125 C.
+
+groovyscript.wiki.prodigytech.ore_refinery.title=Ore Refinery
+groovyscript.wiki.prodigytech.ore_refinery.description=Ore processing machine with 1 input, 2 outputs and an optional chance for the 2nd output.
+groovyscript.wiki.prodigytech.ore_refinery.time.value=The time needed to perform the recipe, in ticks. Will run faster than this number when heated over 125 C.
+groovyscript.wiki.prodigytech.ore_refinery.secondaryChance.value=Sets the chance to get the 2nd output.
+
+groovyscript.wiki.prodigytech.primordialis_reactor.title=Primordialis Reactor
+groovyscript.wiki.prodigytech.primordialis_reactor.description=Turns organic matter into Primordium.
+groovyscript.wiki.prodigytech.primordialis_reactor.addRecipe=Adds an item that can be converted.
```

Comment:
methods got renamed
```
groovyscript.wiki.prodigytech.primordialis_reactor.add
groovyscript.wiki.prodigytech.primordialis_reactor.remove
```

Reply:
intended since those arent really recipes. could do `addItem`/ `removeItem`

Reply:
my point being that when you generate the wiki `groovyscript.wiki.prodigytech.primordialis_reactor.add` is the lang key checked, which doesnt exist. these lang keys should get renamed so they match the method names.